ipaserver/install/server/upgrade.py

```diff
@@ -1375,7 +1375,7 @@ def remove_ds_ra_cert(subject_base):
 def fix_trust_flags():
     root_logger.info('[Fixing trust flags in %s]' % paths.HTTPD_ALIAS_DIR)
 
-    if sysupgrade.get_upgrade_state('http', 'fix_trust_flags'):
+    if sysupgrade.get_upgrade_state('http', 'fix_trust_flags2'):
```

Comment:
I don't understand why you are changing the name of the state rather than adding a new state.